WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
> Stop pretending to be dense: you know exactly what I want to change and what changes I want to make on this file.

Can you tell us that then, if it so obvious? Let us discuss this seriously. Present all you think in an issue or a post. Not through this nuclear bomb diplomacy.

> I want to fix the mistakes made in the Mercenaries file.

I'm not aware of any such.

> I want to change the names to something that makes sense. 

Look at our localisation file.

> The Mercenary file was hastily put together 

Wrong, this is your make-believe.

> You should be thanking me for fixing it, not blocking a very beneficial improvement.

Nothing shall be blocked, nothing shall be thanked. You can explain elsewhere your desired changes and they will evolve from that.